public/app/features/dashboard-scene/scene/ScopesScene.tsx

```diff
@@ -27,11 +27,11 @@ export class ScopesScene extends SceneObjectBase<ScopesSceneState> {
     });
 
     this.addActivationHandler(() => {
-      this.state.filters.fetchScopes();
+      this.state.filters.fetchBaseNodes();
 
       const filtersValueSubscription = this.state.filters.subscribeToState((newState, prevState) => {
-        if (newState.value !== prevState.value) {
-          this.state.dashboards.fetchDashboards(newState.value);
+        if (newState.scopes !== prevState.scopes) {
+          this.state.dashboards.fetchDashboards(newState.scopes);
           sceneGraph.getTimeRange(this.parent!).onRefresh();
         }
       });
```

Comment:
Is the long term plan to have the UX improved for the initial scopes loading? Currently with scopes preserved in the URL I cannot reason about them when opening the scopes selector for the first time, as we only fetch top leel scopes:

<img width="293" alt="image" src="https://github.com/grafana/grafana/assets/2376619/8f976d18-39bf-4a85-8330-f936c08c7bde">


Would be great if we could figure out which sub trees to open based on the persisted scopes.

Reply:
We had a discussion around this yesterday (get the entire path for a given scope name) but that's something that requires a design doc and more thinking on how to do it. For the time being we agreed on being OK with the current mechanism.

Reply:
great!